Makefile

```diff
@@ -32,16 +32,16 @@ format: fmt
 fmt: format-go tidy-go vet-go
 vet: vet-go
 
-lint:  ; $(info $(M) running linters...)
-	@golangci-lint run
+lint:  ; $(info running linters...)
+	@golangci-lint run --config=./.golangci.yaml ./...
```

Comment:
I think this is not necessary.

Reply:
It is not necessary as this is the default. I don't think making it explicit hurts